src/base/utils/foreignapps.cpp

```diff
@@ -129,30 +134,23 @@ namespace
     {
         QString result;
 
-        DWORD type = 0;
         DWORD cbData = 0;
         LPWSTR lpValueName = NULL;
+        std::wstring lpValueNameBuffer;
```

Comment:
Please stop use `lp` prefix with non-LP\* prefixed types. `valueName` is good enough name for it, IMO.

Reply:
`Buffer` suffix seems to be redundant in most the cases here.

Reply:
Fixed